gvsbuild/projects/portaudio.py

```diff
@@ -26,30 +26,18 @@ def __init__(self):
         Project.__init__(
             self,
             "portaudio",
-            archive_url="http://www.portaudio.com/archives/pa_stable_v190600_20161030.tgz",
+            archive_url="http://files.portaudio.com/archives/pa_stable_v190700_20210406.tgz",
+            hash="47efbf42c77c19a05d22e627d42873e991ec0c1357219c0d74ce6a2948cb2def",
             dependencies=[
                 "cmake",
                 "ninja",
             ],
-            patches=[
-                "0001-Do-not-add-suffice-to-the-library-name.patch",
-                "0001-Fix-MSVC-check.patch",
-            ],
         )
 
     def build(self):
         CmakeProject.build(
             self,
             cmake_params="-DPA_DLL_LINK_WITH_STATIC_RUNTIME=off",
             use_ninja=True,
-            do_install=False,
             out_of_source=False,
         )
-
-        self.install(r"portaudio.dll bin")
-        self.install(r"portaudio.pdb bin")
-        self.install(r"portaudio.lib lib")
-
-        self.install(r".\include\* include")
-
-        self.install(r".\LICENSE.txt share\doc\portaudio")
```

Comment:
you should keep the license

Reply:
The CMakelist is already installing it to the correct location, I built it and verified that it was installed in the correct location.